ext-apiserver/pkg/apis/cluster/v1alpha1/v1alpha1_suite_test.go

```diff
@@ -37,6 +37,12 @@ func TestV1alpha1(t *testing.T) {
 	t.Run("crudAccessToMachineClient", func(t *testing.T) {
 		crudAccessToMachineClient(t, cs)
 	})
+	t.Run("crudAccessToMachineSetClient", func(t *testing.T) {
+		// TODO: the following test fails with:
+		// the namespace of the provided object does not match the namespace sent on the request
+		// uncomment when fixed
+		//crudAccessToMachineSetClient(t, cs)
```

Comment:
this is very odd, especially that the machines tests work.

Reply:
I think all the current machine/cluster resource assume default namespace. Is this the case with MachineSet?

Reply:
I think it's not the case, I haven't changed anything in that space. I have just added this piece of code that executes existing test. I think there must be some mismatch between `Machine` and `MachineSet` regarding setting default namespace.